unit-tests/src/test/scala/java/time/DurationSuite.scala

```diff
@@ -0,0 +1,24 @@
+package java.time
+
+object DurationSuite extends tests.Suite {
+
+  val one   = Duration.ofSeconds(350070L, 112233L)
```

Comment:
Are the names `one two three` from the original code or can they be changed?

When I read the code and tried to follow what was going on, I expected `one` to refer to 
one second. Yeah, a leap on my part.  It took me time and doing the math in the declaration
to figure out that it meant something like dataPointOne  By the time I got to `three` I had figured
out the pattern/intent, but lost my humor.  OK, I'm a slow learner ;-)

Reply:
LOL, yeah I did that so it is like a list and easy to reference but `v1, v2 ...` would be better for value1 etc.